### PR TITLE
feat: create application deploy guide using navigation buttons

### PR DESCRIPTION
**Pull request by thlmenezes**
## Description of change
Cria vínculos duplos entre posts permitindo acesso no formato de guia para deploy de aplicação

## Issues resolved by this PR

- [x] [DVN-10215]
- [x] [DVN-10216]

<details>
  <summary>More info to help validate your PR: links, images, videos, ...</summary>

  Uso do botão next

https://user-images.githubusercontent.com/33532177/236733980-b00cd3c2-2229-4c12-9468-b50b3f91a2b8.mp4

Uso do botão previous

https://user-images.githubusercontent.com/33532177/236733991-038ed366-f751-4ac2-81fd-1f0779960f4e.mp4

</details>

[DVN-10215]: https://devopness.atlassian.net/browse/DVN-10215?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[DVN-10216]: https://devopness.atlassian.net/browse/DVN-10216?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: docs/applications/add-application.md [hunk 1]
```diff
@@ -5,7 +5,9 @@ links:
     overview:
     quickstart:
     previous:
+        - /docs/ssh-keys/test-ssh-connection
     next:
+        - /docs/pipelines/add-new-step
```
Comment (jfoliveira) [inline docs/applications/add-application.md:10]: `add new` seems a bit redundant here, as the item to be added is always a new item.
I believe `<operation>-<resource type>` could be a simplified naming convention and it is already adopted in most of the existing docs files:
ted for 
```suggestion
        - /docs/pipelines/add-pipeline-step
```

File: docs/applications/deploy-application.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
+---
+title: Deploy application
+intro:
+links:
+    overview:
+    quickstart:
+    previous:
+        - /docs/configuration-files/create-dotenv-file
```
Comment (jfoliveira) [inline docs/applications/deploy-application.md:8]: 1) I see you're using the terms `add` and `create` interchangeably. Can we stick to `add` as this is the terminology adopted on Devopness web app? 

2) `DotEnv` is only one of the multiple file formats that can be managed via Devopness. Beyond `.env` users can add `.yaml`, `.txt`, `.md`, `.<anything>`.

I believe the documentation topics should be agnostic and re-usable across a multitude of data variation. If any, `DotEnv` can be mentioned in the configuration files steps as an example, but I don't think having a dedicated documentation topic just for `.env` files is needed:
```suggestion
        - /docs/configuration-files/add-configuration-file
```

File: docs/ssh-keys/test-ssh-connection.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
+---
+title: Test SSH connection
```
Comment (jfoliveira) [inline docs/ssh-keys/test-ssh-connection.md:2]: We should guide our users beyond only testing connections or SSH keys. Our users need to effectively connect to their servers using SSH and use their servers the way they want.

I believe we actually need a topic `servers/ssh-into-server` and, if needed, on `ssh-keys/add-ssh-keys` we could link it as a next step.

What do you think?

File: docs/configuration-files/add-configuration-file.md [hunk 4]
```diff
@@ -0,0 +1,14 @@
+---
+title: Create configuration file
```
Comment (jfoliveira) [inline docs/configuration-files/add-configuration-file.md:2]: The file name is prefixed with `add` operation, here title mentions `Create` operation.
Perhaps adopt a single terminology to make it consistent for the reader?

Comment (thlmenezes) [reply docs/configuration-files/add-configuration-file.md:2]:
```suggestion
title: Add configuration file
```

**Review by jfoliveira — APPROVED** [review]

**Pull request merged**